### ISSUE TITLE
[tab] tab button gain forced focus at loading

### ISSUE THREAD

**Issue by georges-gomes** [ing-bank/lion#587]
## Describe the bug

The selected tab button gains focus.

## To Reproduce

If the tab is down in a page, the page will scroll to reach the button - at load.

## Expected behavior

Button should be selected but not gain focus

**Comment by daKmoR** [comment]
hey there 🤗 

I am not sure what you mean? e.g. you load the page and the tab is focused? or do you change the `activeIndex` and then it becomes focused?

I can't seem to reproduce it 🙈 do you maybe have a gif or so?

**Comment by georges-gomes** [comment]
I did used "selectedIndex" attribute but I quickly tested without it and got the same result. Maybe badly tested.

I fixed it quick and dirty with 
```
const selectButton = element => {
  //element.focus();
  element.setAttribute('selected', true);
  element.setAttribute('aria-selected', true);
  element.setAttribute('tabindex', 0);
};
```

Because I think the `selectedIndex` attribute does call `__updateSelected()` which calls `selectButton`.

My fix is aggressive and works for my case but it should more subtle like only focusing on 
`set selectedIndex(value)` but not when first landed on the page.

Hope this helps
